### PR TITLE
Rewritten all DAO using JDBC

### PR DESCRIPTION
**Pull request by Oleh888**

### CODE REVIEW

File: src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            statement.executeUpdate();
+            return true;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete order with id = " + id, e);
+        }
+    }
+
+    private void addToOrder(Order order) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java:98]:
```suggestion
    private void addProductsToOrder(Order order) {
```

File: src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            for (Product product : order.getProducts()) {
+                String query = "INSERT INTO orders_products(order_id, product_id) values(?, ?)";
+                PreparedStatement statement = connection.prepareStatement(query);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java:102]: let's create statement before loop

File: src/main/java/mate/academy/model/Role.java [hunk 3]
```diff
@@ -6,6 +6,11 @@
 
     private Role(RoleName roleName) {
         this.roleName = roleName;
+        if (roleName == RoleName.ADMIN) {
+            id = 1L;
+        } else {
+            id = 2L;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/model/Role.java:13]: what if we will have at least 10 roles?

File: src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java (hunk 2 above)
Comment (Oleh888) [reply src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java:102]: done

File: src/main/java/mate/academy/model/Role.java (hunk 3 above)
Comment (Oleh888) [reply src/main/java/mate/academy/model/Role.java:13]: done

File: src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    @Override
+    public boolean delete(Long id) {
+        String query = "DELETE FROM orders WHERE order_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            statement.executeUpdate();
+            return true;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete order with id = " + id, e);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java:94]: should we delete products from it first?

Comment (Oleh888) [reply src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java]: Agree with you, I should delete info about this order in a table orders_products too

Comment (Oleh888) [reply src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java]: done

**Pull request merged**